share/spice/product_hunt/product_hunt.js

```diff
@@ -47,7 +58,9 @@
                     footer: Spice.product_hunt.footer
                 },
                 variants: {
-                    tileTitle: '2line-small',
+                    tile: 'basic1',
```

Comment:
Sorry about the confusion on this stuff, seriously the docs are coming, we have someone writing them right now. The idea behind `tile: basic(1/2/3/4)` is that it's similar to using `group: text` -- it sets some defaults for the `tileTitle`, `tileSnippet` and `tileFooter` -- if we're explicitly setting them we don't need the `tile: 'Basic1'`